src/azure-cli/azure/cli/command_modules/vm/tests/latest/test_vm_commands.py

```diff
@@ -5086,11 +5086,12 @@ def test_gallery_soft_delete(self, resource_group):
             self.check('softDeletePolicy.isSoftDeleteEnabled', True)
         ])
 
-        self.cmd('sig show -g {rg} -r {gallery_name}', checks=[
+        self.cmd('sig show -g {rg} -r {gallery_name} --expand sharingProfile/Groups', checks=[
```

Comment:
I recommend using the independent parameter `--sharing-groups` instead of directly asking users to enter `--expand sharingProfile/Groups`, because the meaning of this parameter is clearer and more convenient for users to input.  @kangsun-ctrl What do you think of it?

Reply:
sounds good!